profile/README.md

```diff
@@ -1,5 +1,5 @@
-Nexus is a protocol for arbitrary length operations. These operations are modeled using a predefined set of synchronous RPCs.
+Nexus is a protocol for arbitrary duration RPC operations. These operations are modeled using a predefined set of synchronous RPCs.
 
-A Nexus caller sends a request to a Nexus server. The request is handled by a Nexus handler. The handler may respond immediately or return a reference to an asynchronous operation. The caller can cancel an asynchronous operation, check for its outcome, or fetch its current state. The caller can also specify a callback URL that the handler will use to asynchronously deliver the result of an operation when it is complete.
+A Nexus caller sends a request to a Nexus server. The request is handled by a Nexus handler. The handler responds either with a result, or with a reference to an asynchronous operation. The caller can cancel an asynchronous operation, check for its outcome, or fetch its current state. The caller can also specify a callback URL that the handler will use to asynchronously deliver the result of an operation when it is complete.
 
-Nexus is currently in development. For more information, see [the proposal](https://github.com/temporalio/proposals/tree/master/nexus#readme).
+Nexus is currently in development.
```

Comment:
I deleted the link to the proposal because I believe it's unhelpful now due to being out of date.

Reply:
Alternate place(s) to learn more?

Reply:
@dandavison would link to https://docs.temporal.io/nexus which shows the current use of Nexus RPC in the context of Temporal Nexus.